cachi2/core/package_managers/pip.py

```diff
@@ -1648,9 +1666,9 @@ def _validate_provided_hashes(requirements: list[PipRequirement], require_hashes
             )
 
         for hash_spec in hashes:
-            algorithm, _, digest = hash_spec.partition(":")
+            _, _, digest = hash_spec.partition(":")
             if not digest:
-                msg = f"Not a valid hash specifier: {hash_spec!r} (expected algorithm:digest)"
+                msg = f"Not a valid hash specifier: {hash_spec!r} (expected 'algorithm:digest')"
```

Comment:
You added the extra quotes, so now some of the tests fail, either update the tests or drop this hunk.

Reply:
Fixed